src/adapters/poi/bragi_poi.js

```diff
@@ -48,19 +38,29 @@ export default class BragiPoi extends Poi {
         name = label;
       } else {
         name = label.slice(0, splitPosition);
-        alternativeName = label.slice(splitPosition + 1);
       }
     }
     }
 
-    super(id, name, alternativeName, type, {
+    super(id, name, type, {
       lat: feature.geometry.coordinates[1],
       lng: feature.geometry.coordinates[0],
     }, poiClassText, poiSubclassText, geocodingProps.bbox);
 
     this.value = label;
-
     this.queryContext = queryContext;
+    this.address = {
+      name: geocodingProps.address?.name || geocodingProps?.name,
+      city: this._findAdmin(geocodingProps, 'city')?.name,
+      country: this._findAdmin(geocodingProps, 'country')?.name,
+      label: geocodingProps.label,
```

Comment:
I may be wrong, but it looks like you removed every use of an address label. Is it still needed?

Reply:
address.label is used in `src/components/PoiItem`. This way the behavior is changed only for suggestions.
A label contain the zip code, it can be `7 r Jean Lantier, 75001 Paris` for instance.
I'm not sure we can reconstruct this label from name + zip + city, as order might depend from countries ?

Reply:
Ok for now, let's keep it until we see clearer in all this formatting, there are already a lot of improvements.